src/PhpGenerator/Printer.php

```diff
@@ -102,10 +102,12 @@ public function printClass(ClassType $class, PhpNamespace $namespace = null): st
 		}
 
 		$properties = [];
+		$initLengthDefault = strlen($this->indentation) + 4;
```

Comment:
Magic constant `+4` means the length of the ` = ` between the definition and the value and length of the closing `;`